components/slider/index.css

```diff
@@ -10,63 +10,94 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-slider.css";
+
+
+/* missing tokens for variant with Tick, also need t-shirt sizing */
+.spectrum--medium {
+  --spectrum-slider-tick-mark-height: 10px;
+  /* --spectrum-global-dimension-size-125 */
+  --spectrum-slider-tick-mark-width: 2px;
```

Comment:
This appears the same as **slider-track-thickness.** I think we could use the same token here.

Reply:
I think we should keep it as an a var as we will replace those hard-coded values soon with tokens.